bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/controllers/AuthController.java

```diff
@@ -51,7 +55,18 @@ public ResponseEntity<Void> signup(
 			@RequestBody SignUpRequest payload
 	) throws SQLException {
 		int userId = 0;
-		// ... do logic here and get the correct user id
+
+		Instant now = Instant.now();
+
+		UserWithoutId userWithoutId = new UserWithoutId(
+				payload.getFirstName(),
+				payload.getLastName(),
+				payload.getEmail(),
+				payload.getPassword(),
+				payload.getCreatedAt()
```

Comment:
This should be the `now` variable you defined above, not `payload.getCreatedAt`. Remember we want all of our times to be calculated server-side, so there's no need for the user to supply it through the payload